lpms.go

```diff
@@ -65,10 +65,10 @@ func (l *LPMS) Start() error {
 //HandleRTMPPublish offload to the video listener
 func (l *LPMS) HandleRTMPPublish(
 	getStreamID func(reqPath string) (string, error),
-	stream func(reqPath string) (stream.Stream, error),
+	getStream func(reqPath string) (stream.Stream, stream.Stream, error),
```

Comment:
getStream returns 2 streams, one for RTMP, and one for HLS (which the segmenter will populate).  This code change means the HLS stream will always be ready when a RTMP stream is published to the node. (But it won't be sent through the network unless requested.)